### PR TITLE
Asterisk and Chibi updates

### PR DESCRIPTION
**Pull request by Colin-Tel**
## About the PR

Adds required lavaland additions, also Asterisk gets additional meteor shielding around MG's office

## Media

Chibi:
image
Asterisk:
image

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

Not this time

### CODE REVIEW

File: Resources/Maps/asterisk.yml [hunk 1]
```diff
@@ -3543,6 +3547,7 @@ entities:
     - type: MovedGrids
     - type: Broadphase
     - type: OccluderTree
+    - type: LoadedMap
```
Comment (IamVelcroboy) [inline Resources/Maps/asterisk.yml:3550]: Whaz this?

Comment (IamVelcroboy) [reply Resources/Maps/asterisk.yml:3550]: NM, I guess it's normal

Comment (Colin-Tel) [reply Resources/Maps/asterisk.yml:3550]: I don't actually know, I haven't really done anything different as far as running `mapping 69 /maps/asterisk.yml`

Comment (IamVelcroboy) [reply Resources/Maps/asterisk.yml:3550]: All good. All the other maps have it. Just odd that it's getting added here. 🤷‍♂️

**Comment by deltanedas** [comment]
sus

**Comment by Colin-Tel** [comment]
whoops

**Review by IamVelcroboy — APPROVED** [review]

**Pull request merged**